src/SFA.DAS.EmploymentCheck.Functions/Application/Clients/EmploymentCheck/EmploymentCheckClient.cs

```diff
@@ -33,6 +36,14 @@ public async Task<IList<Models.EmploymentCheck>> GetEmploymentChecksBatch()
             return employmentCheckRequests;
         }
 
+        public async Task<IList<EmploymentCheckCacheRequest>> SetEmploymentCheckCacheRequestRelatedRequestsRequestProcessingStatus(
+            Tuple<EmploymentCheckCacheRequest, ProcessingCompletionStatus> employmentCheckCacheRequestAndStatusToSet)
+        {
+            Guard.Against.Null(employmentCheckCacheRequestAndStatusToSet, nameof(employmentCheckCacheRequestAndStatusToSet));
```

Comment:
is this guard necessary? is there a possibility of getting a `null` input? if so, the should be a unit test explaining the scenario and covering this line.